css-four-column-layout/style.css

```diff
@@ -0,0 +1,59 @@
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  font-family: 'Barlow', sans-serif;
+  background-color: rgb(243, 242, 242);
+}
+
+h3,
+p {
+  display: flex;
+  justify-content: center;
+  line-height: 2;
+}
+
+.image > img {
+  width: 100%;
+  background-position: center;
+}
+
+.visual {
+  display: flex;
+  justify-content: center;
+}
+
+.visual > img {
+  width: 70px;
+}
+
+.title {
+  display: flex;
+  justify-content: center;
+}
+
+.row {
+  flex-basis: calc(100% / 4);
+  display: flex;
+  flex-direction: row;
+}
+
+.column-one {
+  width: 25%;
+}
+
+.column-two {
+  width: 25%;
+}
+
+.column-three {
+  width: 25%;
+}
+
+.column-four {
+  width: 25%;
+}
```

Comment:
Since you finished early, how bout a best practice?
```
.column {
  width: 25%
}
```
You can use one class for multiple elements.